fury/actor.py

```diff
@@ -680,8 +686,7 @@ def vtkShaderCallback(_caller, _event, calldata=None):
                                 vtkShaderCallback)
 
     # Color Scale with a lookup table
-    if is_colormap:
-
+    if is_colormap is not None and is_colormap:
```

Comment:
same as above

Reply:
done (I always forget that None is a "false" statement)